data/campaigns/Eastern_Invasion/scenarios/17b_All-In.cfg

```diff
@@ -70,36 +70,21 @@
     [store_gold]
         side={SIDE}
     [/store_gold]
-    [if]
-        [lua]
-            code=<< return wesnoth.current.turn>15 >>
-        [/lua]
-
-        {VARIABLE_CONDITIONAL gold greater_than 150}
-
+    [if] {VARIABLE_CONDITIONAL turn_number greater_than 10}
```

Comment:
Seems some indentation issues in lines 73 and 74

Reply:
Ah, it's always the indentation! Should be fixed now, thanks.